File: app/src/main/java/com/organizobra_mobile/ui/include/IncludeViewModel.java
package com.organizobra_mobile.ui.include;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class IncludeViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public IncludeViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is gallery fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/organizobra_mobile/ui/include/IncludeViewModel.kt
package com.organizobra_mobile.ui.include

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class IncludeViewModel : ViewModel() {
    private val mText = MutableLiveData<String>()

    init {
        mText.value = "This is gallery fragment"
    }

    val text: LiveData<String>
        get() = mText
}